static/app/components/actions/ignore.tsx

```diff
@@ -119,12 +123,20 @@ const IgnoreActions = ({
         ...IGNORE_DURATIONS.map(duration => ({
           key: `for-${duration}`,
           label: <Duration seconds={duration * 60} />,
-          onAction: () => onIgnore({ignoreDuration: duration}),
+          disabled: !issueSupports(group.issueCategory, 'ignoreFor').enabled,
```

Comment:
Btw I feel like the readability for this is a bit weird now since we have to check for `.enabled` when calling `issueSupports`, imo `issueSupports` should just return a boolean and then we have a separate method to get the `disabledReason`

Reply:
Yeah I agree, I added the disabled reason stuff hastily and it could use a more appropriately named function.

Related to that thought, might we want to add the disabled tooltips here as well? Or perhaps in a followup PR.

Reply:
For sure, I was aiming to add those tooltips in a followup PR 👍